tools/packaging/static-build/cache_components_main.sh

```diff
@@ -30,11 +30,20 @@ cache_firecracker_artifacts() {
 }
 
 cache_kernel_artifacts() {
-	local kernel_tarball_name="kata-static-${KERNEL_FLAVOUR}.tar.xz"
+	local kernel_tarball_name="kata-static-${KERNEL_FLAVOR}.tar.xz"
```

Comment:
Every single job is relying on `KERNEL_FLAVOUR` instead of `KERNEL_FLAVOR`.  Please, keep it as it is and adjust the code below to use `KERNEL_FLAVOUR`.

Reply:
Changed everything to `KERNEL_FLAVOUR`